src/SFA.DAS.ApprenticeCommitments.Api/Controllers/RegistrationController.cs

```diff
@@ -57,5 +58,9 @@ public async Task RegistrationReminderSent(Guid apprenticeId, [FromBody] Registr
         [HttpPost("registrations/{apprenticeId}/firstseen")]
         public async Task RegistrationFirstSeen(Guid apprenticeId, [FromBody] RegistrationFirstSeenRequest request)
             => await _mediator.Send(new RegistrationFirstSeenCommand(apprenticeId, request.SeenOn));
+
+        [HttpPost("registrations/stopped/{commitmentsApprenticeshipId}")]
```

Comment:
Since in the longer term, we will be adding the commitmentsApprenticeshipId to the apprenticeship record. Would the routing here makes sense to call it. 

apprenticeships/{commitmentsApprenticeshipId}/stopped (or use a Patch call)

We obviously need to deal with the fall out of when there is no apprenticeship record (and only a registration record), but this then feels more rest like.

The Registration table is simply a staging table when we have no where to put our data, to start directing calls via it seems to give it too much emphasis whilst the main focus should be the Apprenticeship object